src/Servers/Kestrel/Core/src/HttpProtocols.cs

```diff
@@ -15,7 +16,9 @@ public enum HttpProtocols
         Http1 = 0x1,
         Http2 = 0x2,
         Http1AndHttp2 = Http1 | Http2,
+        [RequiresPreviewFeaturesAttribute]
```

Comment:
Nit: one usually omits the suffix `Attribute` when using attributes

```suggestion
        [RequiresPreviewFeatures]
```